src/selection.coffee

```diff
@@ -404,6 +404,28 @@ class Selection extends Model
     @selectLeft() if @isEmpty() and not @editor.isFoldedAtScreenRow(@cursor.getScreenRow())
     @deleteSelectedText()
 
+  # Deprecated: Use {::deleteToBeginningOfWord} instead.
+  backspaceToBeginningOfWord: ->
```

Comment:
Sorry if I missed this in the other pull request, buy why are these being added as deprecated APIs?